app/views/users/_publishing_from_rss.html.erb

```diff
@@ -51,7 +51,7 @@
     </div>
 
     <%= f.hidden_field :tab, value: @tab %>
-    <% button_text = @user.feed_url.present? ? "Save" : "Submit" %>
+    <% button_text = @user.feed_url.present? ? "Save Feed Settings" : "Submit Feed Settings" %>
```

Comment:
to mimic the other "save" buttons that are self explanatory, also helps in e2e testing